Server/JokesOnYou.Web.Api/DTOs/UserUpdateDTO.cs

```diff
@@ -2,10 +2,11 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using JokesOnYou.Web.Api.Models;
 
 namespace JokesOnYou.Web.Api.DTOs
 {
-    public class UserUpdateDTO
+    public class UserUpdateDTO : User
```

Comment:
This shouldnt inherit from our Identity User since it will then contain password etc...